packages/loot-core/src/client/actions/queries.ts

```diff
@@ -237,6 +237,16 @@ export function deleteGroup(id, transferId?) {
   };
 }
 
+export function applyBudgetTemplatesInGroup(month, categoryIds) {
```

Comment:
does this need to be its own function?  All the other actions are up above for example.  The cleanup one is probably the best example.